### ISSUE TITLE
Create Slax module for MLL

### ISSUE THREAD

**Issue by ivandavidov** [ivandavidov/minimal#179]
It will be a good exercise to create Slax module for MLL. The most recent Slax version is 9.6.4 and it supports everything that should allow MLL to be built out of the box, assuming there is properly configured module.

https://www.slax.org/customize.php

**Comment by TechnologyClassroom** [comment]
Slax is an interesting project!  I was using GRML in production, but Slax could be a good drop-in replacement.

**Comment by ivandavidov** [comment]
I have _proof-of-concept_ that MLL can be built and can run in Slax environment. One note - the default ``apt`` prerequisites were not enough. I had to add ``build-essential`` in order to fix issue with missing GCC headers.

---

TODO:

Create Slax module which contains the following:

* The most recent MLL sources in ``/root/mll``
* All ``apt`` prerequisites preinstalled
* Desktop shortcut for one-click build
* Desktop shortcut for one-click run in QEMU
* Custom MLL wallpaper
---

![Minimal Linux Live in Slax](https://scontent.fsof6-1.fna.fbcdn.net/v/t1.0-9/46648897_2291191650921945_53254519273816064_n.png?_nc_cat=106&_nc_ht=scontent.fsof6-1.fna&oh=83a59fd9693287ede29146336786c8d4&oe=5CAD2524)

**Comment by TechnologyClassroom** [comment]
Making a mll build module would be as simple as running apt on the dependencies and running the `savechanges mll.sb` command.

**Comment by ivandavidov** [comment]
Indeed, the technical part is not difficult. It's the details that are more challenging (yet not that difficult), e.g. wallpaper, shortcuts, testing, etc.

**Comment by TechnologyClassroom** [comment]
I figured out the wallpaper portion.  Do you have a wallpaper ready?

```
echo "Changing to /tmp directory..."
cd /tmp

echo "Downloading wallpaper..."
wget http://location.of/your/img/img.jpg

echo "Overwriting Slax wallpaper..."
mv img.jpg /usr/share/wallpapers/slax_wallpaper.jpg

echo "Changing permissions of wallpaper..."
chmod 664 /usr/share/wallpapers/slax_wallpaper.jpg

echo "Making other changes..."
apt update
apt install -y build-essential

echo "Saving Slax module..."
savechanges mll$(date +%Y%m%d-%H%M).sb

echo "Building custom Slax ISO..."
genslaxiso slaxmll$(date +%Y%m%d-%H%M).iso mll*.sb
```
